spesifikasjoner/afpant/afpant-model/examples/kjoepekontrakt-example.xml

```diff
@@ -1,99 +1,109 @@
 <?xml version="1.0" encoding="utf-8"?>
-<?xml-stylesheet type="text/xsl" href="http://localhost:8000/xslt/dsbm-1.0.0.xslt"?>
-<kjoepekontrakt>
-  <megler id="945811714">
-    <foretaksnavn>DNB Eiendom</foretaksnavn>
-    <anvarligMegler>
-      <navn>Ole Olsen</navn>
-      <epost>ole@megler.no</epost>
-      <telefonDirekte>99223452</telefonDirekte>
-      <telefon>55300455</telefon>
-    </anvarligMegler>
-    <referanse>OPGNR-123</referanse>
-    <oppgjorsavdeling id="912345678">
-      <foretaksnavn>Megleroppjør AS</foretaksnavn>
-    </oppgjorsavdeling>
-  </megler>
-  <salgsobjekt>
-    <salgsoppgave>https://www.eiendomsmegler1.no/bolig/kjoepe-bolig/boliger/bolig/?propertyid=126319694&amp;amp;title=NYTT%20byggetrinn%20p%C3%A5%20toppen%20av%20Reppe.%20Flere%20hj%C3%B8rne-%20og%20utsiktsleiligheter.%20Sol%20og%20utsikt%20mot%20by,%20fjord%20og%20mark.&amp;amp;org=EiendomsMegler%201%20Prosjekt%20Trondheim</salgsoppgave>
-    <nettannonse>https://www.finn.no/realestate/newbuildings/ad.html?finnkode=137493979</nettannonse>
-    <registerenheter>
-      <registerenhet>
-        <matrikkel kommunenavn="Bergen" kommunenummer="1201" gaardsnummer="21" bruksnummer="1614" festenummer="1" seksjonsnummer="2" eiendomsnivaa="F"/>
-        <adresse>
-          <gatenavn>Krinkelkroken 1</gatenavn>
-          <postnummer>5014</postnummer>
-          <poststed>Bergen</poststed>
-        </adresse>
-      </registerenhet>
-      <registerenhet>
-        <matrikkel kommunenavn="Bergen" kommunenummer="1201" gaardsnummer="21" bruksnummer="275" festenummer="0" seksjonsnummer="0" eiendomsnivaa="E"/>
-      </registerenhet>
-      <registerenhet>
-        <borettsandel organisasjonsnummer="987654321" borettslagnavn="Holmåsen borettslag" andelsnummer="4"/>
-      </registerenhet>
-      <registerenhet>
-        <aksjeleilighet organisasjonsnummer="987644311" organisasjonsnavn="Z-Eiendom Bolig AS" leilighetsnummer="92"/>
-        <adresse>
-          <gatenavn>Vikebukta 9B</gatenavn>
-          <postnummer>5014</postnummer>
-          <poststed>Bergen</poststed>
-        </adresse>
-      </registerenhet>
-    </registerenheter>
-  </salgsobjekt>
+<?xml-stylesheet type="text/xsl" href="../xslt/dsve-1.0.0.xslt"?>
+<kjoepekontrakt xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../xsd/dsve-1.0.0.xsd">
+	<avsender id="945811714">
+		<foretaksnavn>DNB Eiendom</foretaksnavn>
+		<referanse>OPGNR-123</referanse>
+	</avsender>
+	<mottaker id="945811714">
```

Comment:
Det er ikke naturlig å ha mottaker av kjøpekontrakten her? Siden det potensielt kan levers til flere banker? Da må man lage en for hver. Det blir også tydelig hvem som er mottaker gjennom Altinn?

Reply:
Enig, mottaker setter man i upload til altinn, det skal ikke stå inne i selve kjøpekontrakten. Den kan som du sier gå til flere

Reply:
Ser ikke endringer i xsd for kjøpekontrakten, forresten. Eller er eg blind?

Reply:
For å få loggført på meglersystemsiden hvem den er sendt til så var tanken å medta mottaker i meldingen - da holder det for oss å logge den faktisk produserte xml'en vi sender. om det ikke skal inngå i meldingen måtte vi holdt styr på mottakerne utenfor meldingene. med mottaker inni meldingen blir den også self contained (= teoretisk bedre sporbarhet i fremtiden om hvem man produserte hvilket innhold til).

også because lazy.

Reply:
> Ser ikke endringer i xsd for kjøpekontrakten, forresten. Eller er eg blind?

spesifikke endringen du tenker på? XML validerer mot XSD i PR'en så langt jeg kan se?

Reply:
Det plager ikke meg at det står banknavn inne i meldingen, men da har man egentlig en kjøpekontrakt per mottaker.. dvs hvis 3 banker er inne i bildet så må dere lage og ta vare på 3 versjoner. Det overlater jeg til dere på megler siden å bli enige om ettersom det ikke har noe å si for meg. Jeg kommer bare til å overse det som kommer som jeg ikke har bruk for.

Reply:
I en eventuell tvistesak vil også megler måtte kunne dokumentere hvem som har mottatt hva, men det blir egentlig et mer prinsipielt spørsmål om mottakerinformasjonen skal lagres utenfor eller inni meldingen tenker jeg.